spec/lib/rake/task_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "erb_lint/rake/task"
```

Comment:
It seems like there's a conflict between `Rake` and `FakeFS`. Rake adds methods to `FileUtils` and then calls them (at least `commands`). It looks like you can work around that this way:
```suggestion
FakeFS.deactivate!
require "erb_lint/rake/task"
FakeFS.activate!
```

Reply:
Fixed. Thanks, the full test suite runs now.